src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -813,6 +819,20 @@ public void onTraversalStart() {
         );
     }
 
+    /**
+     *  Checks to see if the given vcf has already been annotated.
+     *
+     *  No need to annotate again.
+     */
+    static void checkIfAlreadyAnnotated(final VCFHeader vcfHeader) {
+        if (vcfHeader.getOtherHeaderLine("Funcotator") != null) {
+            throw new UserException.BadInput("Given VCF has already been annotated!");
+        }
+        else if (vcfHeader.getOtherHeaderLine("Funcotator Version") != null) {
```

Comment:
These header line keys "Funcotator Version" and "Funcotator" should ideally be declared as `static final` constants somewhere -- possibly in `FuncotatorConstants` or `GATKVCFConstants`